### ISSUE TITLE
Spatial audio drops off way too fast

### ISSUE THREAD

**Issue by ara4n** [matrix-org/thirdroom#358]
You expect to be able to easily hear folks who are vaguely close to you, but instead they rapidly get quieter. There should be an amnesty where folks closer than 20m are all equally loud or something

**Comment by vintprox** [comment]
Yes, metaverses are not exactly loved for realism, but rather for having tools for effortless communication.
